packages/ingest/HttpProviderClient.js

```diff
@@ -19,20 +19,25 @@ const log = require('@cumulus/common/log');
 const isValidHostname = require('is-valid-hostname');
 const { buildURL } = require('@cumulus/common/URLUtils');
 const errors = require('@cumulus/errors');
+const get = require('lodash/get');
+
 const { lookupMimeType, decrypt } = require('./util');
 
 const validateHost = (host) => {
   if (isValidHostname(host) || isIp(host)) return;
 
   throw new TypeError(`provider.host is not a valid hostname or IP: ${host}`);
 };
+const HTTP_REQUEST_TIMEOUT_SECONDS = 5 * 60; // Timeout for simplecrawler's headers response
 
 class HttpProviderClient {
   constructor(providerConfig) {
     this.providerConfig = providerConfig;
     this.protocol = providerConfig.protocol;
     this.host = providerConfig.host;
     this.port = providerConfig.port;
+    this.httpRequestTimeoutSeconds = get(providerConfig,
```

Comment:
This should probably be renamed to something more specific to the crawler, rather than just "request timeout".

Reply:
The problem is we're not -really- crawling anything.   See:

`    c.maxDepth = 1;
`

The distinction, particularly to the end user, is at best confusing external to the provider.  Fair enough here, but less so in the components, as it is essentially *the* timeout for fetching the PDR HTML page.

Reply:
`httpListTimeout`, then?

Reply:
e042a5278